presto-verifier/src/main/java/com/facebook/presto/verifier/framework/AbstractVerification.java

```diff
@@ -173,10 +175,7 @@ public VerificationResult run()
             test = Optional.of(getQueryRewrite(TEST));
 
             // Run control queries
-            if (skipControl) {
-                controlQueryContext.setState(NOT_RUN);
-            }
-            else {
+            if (!skipControl) {
```

Comment:
Why invert and reorder?

Reply:
Rest of the code uses `!skipControl` checks. So just wanted to keep it the same for readability.